build-tools/packages/build-cli/src/lib/git.ts

```diff
@@ -187,16 +187,32 @@ export class Repository {
 	 * @returns An array of all commits between the base and head commits.
 	 */
 	public async revList(baseCommit: string, headCommit: string = "HEAD"): Promise<string[]> {
-		const result = await this.git.raw("rev-list", `${baseCommit}..${headCommit}`);
+		const result = await this.git.raw("rev-list", `${baseCommit}..${headCommit}`, "--reverse");
 		return result
 			.split(/\r?\n/)
 			.filter((value) => value !== null && value !== undefined && value !== "");
 	}
 
+	public async mergeAbort() {
+		const abort = await this.git.merge(["--abort"]);
+		console.log("Abort:", abort.result);
+		return false;
+	}
```

Comment:
Not sure this is worth a function, especially since you end up using `await this.git.merge(["--abort"]);` in other code below.